README.md

```diff
@@ -4,33 +4,82 @@
+install any missing dependencies. If you use conda you might prefer to first install
+the dependencies as described in [Dependencies](#dependencies).
 
-First, you would want to choose the directory where you want the code to be located. You can use `ls` to list the folders in the current directory and then use `cd [directory_path]` to enter the directory you want to host the repository.
+To install without dependencies, run `pip install --no-deps`
 
-Next, you would clone the repository using the following command `git clone https://github.com/Center-for-Women-s-Welfare/SSS.git`. 
-Center-for-Women-s-Welfare/SSS
+### Dependencies
+If you are using `conda` to manage your environment, you may wish to install the
+following packages before installing `sss`:
 
-### Creating the Conda Environment
-Now that you are in the repository folder, you would want to create your conda environment using the sss.yml file. To do so, you would use `conda env create -f sss.yml`.
+Required:
 
-If you want to add or remove a package, add it to the yaml file and then run: `conda env update -f <path_to_yaml_file> --prune`. 
+* pandas
+* numpy
+* setuptools_scm
+* sqlalchemy
 
-If you have a problem and need to start fresh, you can delete the environment using: `conda remove --name <env_name> --all` (then you can re-make it with the first command). 
+If you want to do development on sss, in addition to the other dependencies
+you will need the following packages:
 
-After we have created the environment, you will type `conda activate sss` to ensure we working in the conda environment that contains all the dependencies that were included in the sss.yml file.
+* pytest
+* pytest-cov
+* coverage
+* sphinx
+* pypandoc
 
-### Install Packages
-From there, you want to install the repository as a package, using the command `pip install .`
+One way to ensure you have all the needed packages is to use the included
+`sss.yaml` file to create a new environment that will
+contain all the optional dependencies along with dependencies required for
+testing and development (```conda env create -f sss.yml```).
+Alternatively, you can specify `dev` when installing sss
+(as in `pip install .[dev]`) to install the packages needed for testing
+and documentation development.
+
+## Tests
+Uses the `pytest` package to execute test suite.
+From the source sss directory run ```pytest``` or ```python -m pytest```.
 
 ## Creating the Database
-It is important to understand that **this is meant to be done once**. To create the database we would call `./scripts/create_database.py` in our terminal. The script also takes an optional parameter of the database_url which would be the name of the database. Currently the default is **'sqlite:///sss.sqlite'** `./scripts/create_database.py -d [url name]`
+In normal use, once you create the database, you will not need to do this again.
+When testing, however, you may need to delete and it re-make it.
```

Comment:
Should we link to how to delete (or just add in rm __)

Reply:
I added a parenthetical saying to just delete the file.